commons/src/test/java/io/aiven/kafka/tiered/storage/commons/chunkindex/ChunkIndexSerializationTest.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2023 Aiven Oy
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.aiven.kafka.tiered.storage.commons.chunkindex;
+
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.util.Arrays;
+import java.util.Base64;
+import java.util.List;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.luben.zstd.ZstdCompressCtx;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ChunkIndexSerializationTest {
+    final ObjectMapper mapper = new ObjectMapper();
+
+    static final String ENCODED_CHUNKS = "KLUv/SAPeQAAAAAAAwAAAAoBAAoAAAAe";
+
+    // This should give an idea what's inside ENCODED_CHUNKS.
+    static {
```

Comment:
Should this be a proper test and not a static block?

Reply:
No, this is not a test, but a visualization showing what `KLUv/SAPeQAAAAAAAwAAAAoBAAoAAAAe` really means and how it's obtained. Unfortunately, people who are going to read this code cannot perform Zstd compression in their heads in reasonable time to validate it :)